src/components/learner-credit-management/cards/CourseCard.jsx

```diff
@@ -75,6 +80,15 @@ const CourseCard = ({
 
   const isExecEd = course_type === EXEC_COURSE_TYPE;
 
+  let linkToCourse;
+  if (content_type === 'program') {
+    linkToCourse = `${ENTERPRISE_LEARNER_PORTAL_URL}/${enterpriseSlug}/program/${uuid}`;
```

Comment:
Do we have a unit test for the program scenario?

Reply:
good catch! i've added another test for programs